src/common/gui/CAboutBox.cpp

```diff
@@ -29,6 +35,26 @@ void CAboutBox::draw(CDrawContext* pContext)
    if (value)
    {
       _aboutBitmap->draw(pContext, getViewSize(), CPoint(0, 0), 0xff);
+
+      int strHeight = infoFont->getSize(); // There should really be a better API for this in VSTGUI
+      std::vector< std::string > msgs = { {
+              std::string( "Version 1.6.0 beta (build: " ) + __DATE__ + " " + __TIME__ + ")",
+              "Released under the GNU General Public License, v3",
```

Comment:
Should be a file lets say `VERSION` in root directory where the version would be grabbed?

Reply:
Well we should have a built artifact with all our information! That's why this is an incomplete diff. For now I just hardcoded 1.6 so I didn't have to deal with cross platform resource loads but to properly finish this we need to create the XML file we were discussing which should include the version and so on and then not have this here.

My thinking was "hardcoded as string is better than as png" and "getting the build date and correct license" was valuable which is why I did this slightly but still slightly less tacky interim step.